fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1417,7 +1417,7 @@ createBaselinePlanStatement
     ;
 
 dropBaselinePlanStatement
-    : DROP BASELINE INTEGER_VALUE
```

Comment:
I think we need to provide a syntax that allows users to drop all baseline.

Reply:
may consider this interface. You reminded me of something; I need to add some enable/disable interfaces. I will add it in other PR